packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -64,6 +54,7 @@ export function Previewer({
           </Text>
           <UploadButton
             acceptedFileTypes={acceptedFileTypes}
+            isLoading={isLoading}
```

Comment:
Added a quick bug fix, now the `UploadButton` and `UploadDropZone` will be disabled when the upload is happening. Before, someone could add more files during the download, which was unusual